### ISSUE TITLE
Conditional assignments within a port instantiation break syntax highlighting pattern on the next line

### ISSUE THREAD

**Issue by birdybro** [eirikpre/VSCode-SystemVerilog#188]
https://github.com/MiSTer-devel/SMS_MiSTer/blob/master/SMS.sv#L477-L478

![image](https://user-images.githubusercontent.com/16388068/198046155-4f655a8e-55ed-4555-b647-d65bc022cac0.png)

**Comment by joecrop** [comment]
I'll be honest, this is low on the priority list because must commercial linting tools would scream bloody murder when you put this amount of logic directly in a port instantiation.  An easy fix is to just assign this into a net before passing it to the port.

**Comment by birdybro** [comment]
Agreed on principle for sure, but I figured I'd report it anyways regardless as it was unexpected behavior.

[This extension](https://github.com/mshr-h/vscode-verilog-hdl-support) does handle it correctly, but I prefer yours :P

**Comment by joecrop** [comment]
OK, I finally had time to dig into this and I have some bad... well maybe good? news. I can't recreate the problem in a VSCode environment with just the systemVerilog extension installed.

![image](https://github.com/eirikpre/VSCode-SystemVerilog/assets/3010369/8668b70c-9698-4772-8e8e-c5655a43cd2c)

I wonder if you have another extension installed that is either overriding this extension or conflicting wtih it.

**Comment by birdybro** [comment]
I just disabled the other extension from Masahiro Hiramori, I reinstalled yours, and the issue comes back. Maybe your color scheme youa re using doesn't differentiate between the two? Also is this the current release that is up on the vscode extension store you are using? I'm using the Dracula Pro color scheme which might make a distinction between two elements and yours might treat them as the same color... hrmmm

![image](https://github.com/eirikpre/VSCode-SystemVerilog/assets/16388068/b329b067-9d4e-48bc-b72a-528216bf8b19)

I used the file from the time I made this issue --> https://github.com/MiSTer-devel/SMS_MiSTer/blob/901ff4b5683d12929614c4250d13c4ca4e8b62b0/SMS.sv

Using the latest version makes no difference, just wanted to make sure there wasn't a change in code that somehow altered this.

Just confirmed, I changed to Dark Modern (not part of the Dracula color scheme) and sure enough they are the same color now. So whatever special distinction dracula makes between these two is not represented in probably a lot of other color schemes. Not a false positive, but probably not a thing most people would see unless they happened to use your extension combined with using Dracula color themes.

I don't know how to debug linters to see how the output differs to the perspective of how the color scheme is applied to different elements, or else I would help provide more info.

https://draculatheme.com/visual-studio-code - Here is the theme to recreate the issue.

**Comment by birdybro** [comment]
Thank you!
